src/app/util/ember-print.cpp

```diff
@@ -48,35 +49,38 @@ void emberAfPrintln(int category, const char * format, ...)
     }
 }
 
-// TODO: issue #3662 - Unbounded stack in emberAfPrintBuffer()
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wstack-usage="
-
 void emberAfPrintBuffer(int category, const uint8_t * buffer, uint16_t length, bool withSpace)
```

Comment:
Is there a way we could unit test this? Maybe add a TODO PR (I think I wanted to do app/util unit tests before and failed, so it does not seem fair to ask you do do it as part of this PR).

Reply:
Added TODO. I did verify the change out-of-tree with a `#define emberAfPrint(c,...) printf(__VA_ARGS__)`.